src/codacy-coffeescript.coffee

```diff
@@ -7,6 +7,11 @@ try
 catch
     userConfig = {}
 
+for tool in userConfig.tools when tool.name is 'coffeelint'
+    for pattern in tool.patterns
+        for parameter in pattern.parameters when parameter.name is 'value'
+            lintConfig[pattern.patternId]["value"] = parameter.value
```

Comment:
Assumes the values in .codacy are in the expected type (String, Integer, Boolean, etc)